api-reference/50 Common/utils/excelExporter/exportDataGrid(options).md

```diff
@@ -14,7 +14,7 @@ Export settings.
 
 ---
      
-This method requires <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> to export data and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> to save files.
+This method requires <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> to export data and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> to save files. The recommended version of **ExcelJS** is `3.3.1`. When used in Angular, the recommended version is `1.7.0`.
```

Comment:
Whether or not it's better to provide a link to exceljs github issue right in this text?

Reply:
So... 
Our internal research at T847450 points to https://github.com/exceljs/exceljs/issues/984 and it's better to keep a reference to exceljs issue in addition to providing our solution.
What do you think?

Reply:
I'm not really sure about it since I don't remember any links to issues in our documentation. I understand that I might have missed them. It seems to me that we should wait for @RomanTsukanov

Reply:
I would refrain from linking issues in the docs.

Aren't the recommendations about the version too strict? As a user, I would expect a range of supported versions, not just one. And why is the recommended version for Angular so old?

Reply:
ok, we can to support our export without this link in our docs.

Version: we researched how our export works in Angular apps in T847450 and found that there are no issues with ExcelJS v1.7.0. It was released at Jan 2019. Issue with Angular apps is not resolved event in latest versions, there are several WA: [exceljs/exceljs#984](https://github.com/exceljs/exceljs/issues/984).